org.multimc.MultiMC.yml

```diff
@@ -0,0 +1,51 @@
+app-id: org.multimc.MultiMC
+default-branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15'
+command: multimc
+separate-locales: false
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/openjdk11/bin
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - ./install-openjdk11.sh
+    sources:
+      - type: script
+        dest-filename: install-openjdk11.sh
+        commands:
+          - mkdir -p ${FLATPAK_DEST}/jre/openjdk11/bin
+          - cd /usr/lib/sdk/openjdk11/jvm/openjdk-11
+          - cp -ra conf lib release                                          ${FLATPAK_DEST}/jre/openjdk11
+          - rm ${FLATPAK_DEST}/jre/openjdk11/lib/src.zip                     ${FLATPAK_DEST}/jre/openjdk11/lib/ct.sym
+          - cp -ra bin/{java,jjs,keytool,pack200,rmid,rmiregistry,unpack200} ${FLATPAK_DEST}/jre/openjdk11/bin
+  - name: multimc
+    buildsystem: cmake
+    config-opts:
+      - -DMultiMC_LAYOUT=lin-system
+    cleanup:
+      - /lib/debug
+    sources:
+      - type: git
+        url: https://github.com/MultiMC/MultiMC5
+        commit: 3ad9ea507e945709b01a5fcc51a60f7c45ad4c38
+      - type: patch
+        path: java-utils.patch
+      - type: file
+        path: org.multimc.MultiMC.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html